src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -12,11 +12,14 @@ There is a clear lifecycle for all Appwrite Functions, from beginning to end.
 Here's everything that happens during a function execution.
 
 1. The function is invoked.
-2. Appwrite passes in request information like headers, body or path through the `context.req` object.
-3. The runtime executes the code you defined, you can log through the `context.log()` or `context.error()` methods.
-4. Function terminates when you return results using `return context.res.send()`, `return context.res.json()` or similar.
+1. The active [deployment](/docs/products/functions/deploy)'s executor will handle the request.
```

Comment:
These steps are all 1.

Reply:
Yes markdown renders them as 1.2.3.4 ;) This in fact works